Security Changes

FearTheBlackout opened the pull request:
- 2 Security Contractors have been added to the Security department.
- 2 Security Contractor lockers have been added to the Brig.
- The available slots for Forensic Technician have been raised to two.
- Forensic Technicians can now play as Contractors.
- An extra locker has been added to the FT's office.
- Detective has been added as an alternate title for Detective.

Commit:
Security Changes

babydoll commented:
That is 10 slots.

Commit:
Changelog

Zero-Bits commented:
I'm all for security contractors, but yeah, too many security slots.

babydoll commented:
Just open the regular slots to contractors. 4 officer slots. Open FT slot to contractors. 1 detective slot. Yes, you will have to complete for slots **like every other job in the game**.

Commit:
Fixes and changes

Anton-Kr commented:
10 is fine. It was pretty much that many on exodus anyway. If it's too much we can always axe the extra FT and an MA.

Nerev4r commented:
Extra FT isn't at all needed.

PsiOmegaDelta commented on maps/torch/loadout/loadout_suit.dm:
```diff
@@ -35,7 +35,7 @@
 	path = /obj/item/clothing/suit
 	cost = 3
-	allowed_roles = RESTRICTED_ROLES
+	allowed_roles = list(/datum/job/assistant, /datum/job/bartender, /datum/job/officer_contractor, /datum/job/merchant, "Off-Duty",/datum/job/stowaway)
```
"Off-Duty" is not a valid `/datum/job/`subtype.

Minijar commented:
I was under the impression the general consensus in that thread was a big fat NO to this?

I'm opposed to opening up a government ship's security department to a private firm who will only have financial and private interests at heart, rather than the protection of the ship.

Kelenius commented on code/modules/clothing/under/jobs/security.dm:
```diff
@@ -29,7 +29,7 @@
 	item_state = "r_suit"
 	worn_state = "secred"
-	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
What does this have to do with the rest of the changes and why are you sneaking it in?

FearTheBlackout replied:
Security Contractors don't need a tactical edge in combat. Probably better if I axe the armor altogether.

Crushtoe commented:
\>People suggest opening forensics technician as a helpful middle ground.
\>Open up forensics technician _and_ still adds two more contractor slots.
\>Well, at least it's still weaker than Exodus sec, because there's only one detective. (Actual argument point from the thread.
\>Opens up a new forensic technician slot.

In case my sarcasm made this too painful to understand: This is taking almost every single middleground and compromise suggestion and smashing them together to create one mega contractor PR. I am against this; while I would support tentatively opening forensics to contractors, with some major consideration being due, I'm firmly against this massive PR.

Commit:
Changes

Commit:
Changes

Commit:
Changes

Commit:
Changes

Kelenius commented on code/modules/clothing/under/jobs/security.dm:
```diff
@@ -29,8 +29,6 @@
 	icon_state = "security"
 	item_state = "r_suit"
 	worn_state = "secred"
-	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
I repeat the question: why is the armor being removed and why is this change not documented?

FearTheBlackout replied:
Not seeing how this affects anything besides gameplay balance. Security Contractors don't need an undue advantage in combat over their MaA/MP counterparts.

Hubblenaut replied:
All departments get some base protection against their respective damage type, should they have one. Just leave it, this change is unrelated to the point of this PR.
Also :-1: for sneaking this in.

Commit:
Reverts thingy

Kelenius commented on code/modules/clothing/under/jobs/security.dm:
```diff
@@ -79,7 +79,7 @@
 	item_state = "det"
 	worn_state = "detective"
-	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
You're almost there.

Commit:
Reverts other thingy

BlueNexus commented:
I suggest using a PMC/Private Security Company like PCRC (Proxima Centauri Risk Control) for it instead, for the following reasons:

1. PCRC already have their own uniform, helmet, and vest sprites
2. Already present in the lore, so we don't have to make up a bunch of new fluff
3. Used to discipline and following orders
4. Easier to justify them being on the Torch.
5. Would be trained in providing security, and more professional than a random civilian contractor
6. Still not military

![](https://i.imgur.com/GevR3eO.gif)
A PCRC contractor in full gear.

FearTheBlackout commented:
Sounds good. Will work on it during the weekend.

PurpleMartinJCK commented:
Please edit the PR to add only a slot for a contractor forensics tech. At that time it will be reviewed for code quality. A new thread will be made to discuss the other contractor slots. Do not discuss them here.

PurpleMartinJCK commented:
Further discussion of main slots is to go here https://baystation12.net/forums/threads/security-contractors-continued.5884/

ghost commented:
Will work on the above changes in a few days from now when I'm able to. Please leave this open until then.

Techhead0 commented on maps/torch/job/jobs.dm:
```diff
@@ -589,18 +589,21 @@
+	alt_titles = list("Criminal Investigator",
+						"Detective"
+	)
+	outfit_type = /decl/hierarchy/outfit/job/security/detective/forensic
```
As is, EC doesn't get the EC outfit, they're getting the Exodus detective outfit.

Commit:
Removes OA beret

AtomicWorm commented:
Just gonna leave this here: https://en.wikipedia.org/wiki/Monopoly_on_violence

Hopefully my message makes itself clear.

Commit:
Merge remote-tracking branch 'Baystation12/dev' into dev

Commit:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Detective Changes

Commit:
HUD icon

Commit:
Merge remote-tracking branch 'Baystation12/dev' into dev

Commit:
Security Changes


Changelog


Fixes and changes


Merge pull request #19816 from Kearel/deityNewMethod

Reworks deity early game
Merge pull request #19730 from Casper3667/patch-2

Some borg module item changes
Automatic changelog generation for PR #19730 [ci skip]
Dockerize database and gameserver

Adds mining access for NT Pilot (#19829)

* Adding NT pilot the mining access

* Squashed due to author inability to do so.

Multi-Z connectivity no longer limited by bit field (#19832)


Merge pull request #19821 from orelbon/tazers

New taser sprites
Expands hydroponics, adds bees.

Changes
Changes
Changes
Changes
Reverts thingy
Reverts other thingy
Bandaid fix for a runtime when borgs attempt to access their subsystems via the Subsystems tab.

Enforces taj having Siik'maas
Fix mounted flashes permanently confusing ghosts

Automatic changelog update [ci skip]

Restore locker welding functionality

Ports over three IPC screens from Polaris.

Adjusts lights

Increases range a bit
Makes them less tinted, just ever slight yellow tint over urine-in-the-eyes.
Changelog...

...because autolog is beyond me, and I don't really care about it much to begin with

Merge pull request #19840 from xales/docker

Dockerize database and gameserver
Merge pull request #19835 from sabiram/hatfix17dec

Fix #19822 - duplicate definition
Added extra and updated(?) dream text

Merge pull request #19849 from sabiram/weldylocker

Restore locker welding functionality
Automatic changelog generation for PR #19849 [ci skip]
Merge pull request #19859 from Roaper/master

Added extra and updated(?) dream text
Merge pull request #19857 from comma/lightbringer

Adjusts lights
Merge pull request #19846 from sabiram/flash3

Fix mounted flashes permanently confusing ghosts
Automatic changelog generation for PR #19846 [ci skip]
Fixes #19655

Vital organs getting hit by stun_act now make you drop items and Stun for 6 ticks, or Weaken if you go past max_damage of pain.
Capped leg failure chance.
Lowered baton's agony damage and power draw.
Made legs slightly harder to hit.

In effect, it takes 3 hits on chest to get to stunning, 4+ to floor like in ye olde days. Maybe 1 hit less for head (and groin for human/taj hehe)
Leg hits still down in 2ish hits, but not as reliable now since not guaranteed + higher miss chance.
Add toasting mechanics

Sound courtesy of http://www.freesfx.co.uk/

Merge pull request #19858 from sabiram/attentionplease

Add toasting mechanics
Automatic changelog generation for PR #19858 [ci skip]
Merge pull request #19827 from sabiram/carpet16dec

Fix #19824 - maps in some green carpet
Merge pull request #19856 from tlc2013/dev

Ports over three IPC screens from Polaris.
Adds envelops for documents

Resprite of folders, start sealed, gotta break seal to access stuff inside.
Adds mroe sekrit docs to Torch

Makes RD vatties folder envelop too.
Makes nuke instructions go into envelope too

Generalizes it to not always talk about blowing up Torch too.
New -> Init


Corrects the Saber's name

corrects the WT45's empty icon

Added new overmap location - slavers base inside asteroid.

Add unique icons for blast door controllers

Make maintenance drones fast

Ports Eris climbing


Allows lattice, checks if it can pass using right proc. Fixes


Adds Torch mission patch to loadouts

Goes on clothes or spacesuits.
Merge pull request #19223 from CountAlex/slavers_base

Added new overmap location - slavers base inside asteroid.
Moves everything to the helper proc.

Also removes slipping on chair
Automatic changelog update [ci skip]

Fixes heart not stopping bug.

Makes dickwielding a species property

Allows xenos to have them anywhere on the body
Merge pull request #19870 from BlueNexus/gunsforthegungodbulletsforthebulletthrone

Corrects the 9mm machine pistol's name and other fixes
Automatic changelog generation for PR #19870 [ci skip]
Merge pull request #19779 from orelbon/red_alert_vote

Auto crew transfer vote now works during code red
Merge pull request #19878 from FTangSteve/heartStop

Fixes heart not stopping bug.
Merge pull request #19873 from sabiram/buttinbuttoff

Add unique icons for blast door controllers
Merge pull request #19847 from sabiram/speedy

Make maintenance drones faster
Automatic changelog generation for PR #19847 [ci skip]
Merge pull request #19845 from Casper3667/patch-1

Enforces taj having Siik'maas
Automatic changelog generation for PR #19845 [ci skip]
Merge pull request #19841 from Kelenius/Bees

Expands hydroponics, adds bees.
Automatic changelog generation for PR #19841 [ci skip]
destination will be checked before moving to prevent falling


Converts casino ship into actual overmap ship.

Automatic changelog update [ci skip]

Merge remote-tracking branch 'Baystation12/dev' into dev
Merge remote-tracking branch 'origin/dev' into dev


Unathi Buffs: Nutrition and Regeneration

Detective Changes


HUD icon


Added classified project data disk, basically a fluff item that contains some flavor text about the blueprints and designs inside. Gives the antags an interesting item to steal. The CE's locker spawns with five of them.

Grommer


Merge pull request #19875 from CountAlex/casino_ship

Converts casino ship into actual overmap ship
Merge pull request #19879 from comma/swag

Adds Torch mission patch to loadouts
Balances some of the guns


Merge pull request #19886 from orelbon/gun_edits

Tries to balances some of the guns
Automatic changelog generation for PR #19886 [ci skip]
Merge pull request #19885 from ParadoxSpace/gogogodzilla

Unathi Buffs: Nutrition and Regeneration
Makes Ion weapons not shit


Adds helmet covers


Merge pull request #19888 from orelbon/ion_weapons

Makes Ion weapons not shit
Automatic changelog generation for PR #19888 [ci skip]
Merge pull request #19871 from comma/order66

More ""secret"" documents on Torch, sealed envelopes for them
Automatic changelog generation for PR #19871 [ci skip]
Automatic changelog update [ci skip]

Merge pull request #19884 from AtomicWorm/got-you-covered

Colored fabric covers for helmets
Automatic changelog generation for PR #19884 [ci skip]
Merge pull request #19843 from thasc/subsystems-bandaid

Bandaid fix for subsystems access runtime
Automatic changelog update [ci skip]

Unathi healing tweaks/nerfs

Unathi no longer recover oxyloss, ignore healing toggle, or repair dead internal organs.
They also are noticeably less ravenous in their normal hunger.
Fixes #19902
Fixes posibrain construction steps

Generally cut the rate of Unathi healing.

Also, now respects organ_regeneration_multiplier in config.
Fixes Unathi vital organ explosion bug.
Merge pull request #19874 from CrimsonShrike/NoWayButUp

Ports Eris climbing
Merge pull request #19864 from comma/zzop

Fixes stunbatons
Automatic changelog update [ci skip]

Merge pull request #19880 from dryerlint/19-12-2017-sooprsecrit

Added classified project data disk, a fluff item with "secret blueprints and designs" contained inside
Merge pull request #19731 from Hubblenaut/posibrain

Fixes posibrain construction steps
Merge pull request #19903 from Techhead0/fail-scales

Unathi healing tweaks/nerfs
Merge remote-tracking branch 'Baystation12/dev' into dev

Commit:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Map

Commit:
Map

Commit:
Merge pull request #1 from FearTheBlackout/feartheblackout/muhmarinecover

Removes OA beret from loadout

ghost commented:
lol idek anymore

Anton-Kr commented:
RIP DREAMS
